decidim-meetings/app/cells/decidim/meetings/meeting_m_cell.rb

```diff
@@ -8,8 +8,6 @@ class MeetingMCell < Decidim::CardMCell
       include MeetingCellsHelper
 
       def has_authors?
-        # todo
-        # return unless (if the component has the setting enabled)
         true
```

Comment:
I think I don't get this, does it always has authors? Then, why we have to ask?

Reply:
It always has authors, but we have to overwrite this mthod from the parent cell (`Decidim::CardMCell`) so that authors can be rendered! 😄